### PR TITLE
Modify xerox.yaml for DocuPrint M225

### PR DESCRIPTION
**Pull request by wackychocolatefactory**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

**Comment by laf** [comment]
Thanks for this, we also need a unit test as well to cover the example.

https://github.com/librenms/librenms/blob/master/tests/snmpsim/xerox-docuprint.snmprec
https://github.com/librenms/librenms/blob/master/tests/OSDiscoveryTest.php#L1494

**Comment by wackychocolatefactory** [comment]
Sure thing, I'll write them up tomorrow at work. :)

**Comment by wackychocolatefactory** [comment]
Curious, does this one need a new unit test, wouldn't the existing unit test cover the new regex?

**Comment by laf** [comment]
It possibly will but it's more the real world data we want. If detection was changed in the future we would only be able to ensure it matches what we have now and could break things.

File: includes/definitions/xerox.yaml [hunk 1]
```diff
@@ -10,3 +10,4 @@ discovery:
         - Xerox Phaser
         - Xerox WorkCentre
         - FUJI XEROX DocuPrint
+    - sysDescr_regex: '/^.*DocuPrint.*/'
```
Comment (murrant) [inline includes/definitions/xerox.yaml]: This is actually the same as `- sysDescr: DocuPrint`  the sysDescr section searches for the given string anywhere in the sysDescr

**Pull request merged**